TrigScint/src/TrigScint/TrigScintFirmwareTracker.cxx

```diff
@@ -86,7 +97,9 @@ void TrigScintFirmwareTracker::produce(framework::Event &event) {
   Hit HPad2[NHITS];
   Hit HPad3[NHITS];
 
-  Cluster Pad1[NCLUS];
+  // Pad1 goes with NTRK bc of firmware bandwidth constraints
+  // It is also expected on Pad1 to have 1 cluster per track
+  Cluster Pad1[NTRK];
```

Comment:
This I confirmed with Rory that it was a mistake originally

Reply:
Yep :)